docs/tutorials/08-tutorial-sug-roc-notleap.ipynb

```diff
@@ -0,0 +1,981 @@
+      "2023-10-09 02:10:15 PM Generating a residue template for [H][O][C@]1([H])[C@@]([H])([O][H])[C@@]2([H])[O][C@@]3([H])[C@]([H])([O][H])[C@@]([H])([O][H])[C@@]([H])([O][C@@]4([H])[C@]([H])([O][H])[C@@]([H])([O][H])[C@@]([H])([O][C@@]5([H])[C@]([H])([O][H])[C@@]([H])([O][H])[C@@]([H])([O][C@@]6([H])[C@]([H])([O][H])[C@@]([H])([O][H])[C@@]([H])([O][C@@]7([H])[C@]([H])([O][H])[C@@]([H])([O][H])[C@@]([H])([O][C@@]8([H])[C@]([H])([O][H])[C@@]([H])([O][H])[C@@]([H])([O][C@@]9([H])[C@]([H])([O][H])[C@@]([H])([O][H])[C@@]([H])([O][C@]1([H])[C@@]([H])([C]([H])([H])[S][C]([H])([H])[C]([H])([H])[C](=[O])[O-])[O]2)[O][C@]9([H])[C]([H])([H])[S][C]([H])([H])[C]([H])([H])[C](=[O])[O-])[O][C@]8([H])[C]([H])([H])[S][C]([H])([H])[C]([H])([H])[C](=[O])[O-])[O][C@]7([H])[C]([H])([H])[S][C]([H])([H])[C]([H])([H])[C](=[O])[O-])[O][C@]6([H])[C]([H])([H])[S][C]([H])([H])[C]([H])([H])[C](=[O])[O-])[O][C@]5([H])[C]([H])([H])[S][C]([H])([H])[C]([H])([H])[C](=[O])[O-])[O][C@]4([H])[C]([H])([H])[S][C]([H])([H])[C]([H])([H])[C](=[O])[O-])[O][C@]3([H])[C]([H])([H])[S][C]([H])([H])[C]([H])([H])[C](=[O])[O-] using openff-2.0.0\n",
+      "2023-10-09 02:10:16 PM Requested to generate parameters for residue <Residue 1 (ROC) of chain 1>\n",
+      "2023-10-09 02:10:16 PM Generating a residue template for [H][O][C@@]1([H])[C]([H])([H])[C@]2([H])[C]([H])([H])[C]([H])([H])[C@]3([H])[C@]([H])([C]([H])([H])[C]([H])([H])[C@]4([C]([H])([H])[H])[C@@]([H])([O][C](=[O])[C]([H])([H])[H])[C@@]([H])([N+]5([C]([H])([H])[C]([H])=[C]([H])[H])[C]([H])([H])[C]([H])([H])[C]([H])([H])[C]5([H])[H])[C]([H])([H])[C@@]34[H])[C@@]2([C]([H])([H])[H])[C]([H])([H])[C@]1([H])[N+]1([H])[C]([H])([H])[C]([H])([H])[O][C]([H])([H])[C]1([H])[H] using openff-2.0.0\n"
+     ]
+    }
+   ],
+   "source": [
+    "pdb = PDBFile(f\"{datadir}/sug_roc.pdb\")\n",
+    "model = Modeller(topology.to_openmm(ensure_unique_atom_names=True), pdb.positions)\n",
+    "unit_cell_dimensions = Vec3(5.0, 5.0, 10.0) * nanometers\n",
+    "model.topology.setUnitCellDimensions(unit_cell_dimensions)\n",
+    "\n",
+    "system = ff.createSystem(\n",
+    "    model.topology,\n",
+    "    nonbondedMethod=PME,\n",
+    "    nonbondedCutoff=0.9 * nanometers,\n",
+    "    rigidWater=True,\n",
+    "    constraints=HBonds,\n",
+    "    hydrogenMass=3.02352 * amu,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2f7739c7-11ca-4a0f-911e-50a41d2ffd9a",
+   "metadata": {},
+   "source": [
+    "#### Use pAPRika to align the complex such that the guest is along the z axis. The host ring will lie in the xy plane."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "21fc58f4-c6fe-4555-9022-f8d3739cb8dd",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "2023-10-09 02:10:16 PM Note: detected 128 virtual cores but NumExpr set to maximum of 64, check \"NUMEXPR_MAX_THREADS\" environment variable.\n",
```

Comment:
Do you know a way to suppress these warning messages in Notebook? Not critical but it is a bit annoying.

Reply:
I add the line `os.environ['NUMEXPR_MAX_THREADS'] = 'n'` on my machine to get it to shut up, but I don't know what library is dependent on `numexpr` and causing this message. Is this caused by a pAPRika dependency?